core/src/main/scala/org/apache/pekko/persistence/jdbc/state/SequenceNextValUpdater.scala

```diff
@@ -60,3 +60,31 @@ import slick.sql.SqlStreamingAction
 
   def getSequenceNextValueExpr() = sql"""#$nextValFetcher""".as[String]
 }
+
+/**
+ * INTERNAL API
+ */
+@InternalApi private[jdbc] class SqlServerSequenceNextValUpdater(profile: JdbcProfile,
+    val durableStateTableCfg: DurableStateTableConfiguration)
+    extends SequenceNextValUpdater {
+
+  import profile.api._
+
+  final val nextValFetcher = s"""SELECT NEXT VALUE FOR ${durableStateTableCfg.columnNames.globalOffset}""".stripMargin
+
+  def getSequenceNextValueExpr() = sql"""#$nextValFetcher""".as[String]
+}
+
+/**
+ * INTERNAL API
+ */
+@InternalApi private[jdbc] class OracleSequenceNextValUpdater(profile: JdbcProfile,
+    val durableStateTableCfg: DurableStateTableConfiguration)
```

Comment:
`val` is not needed.

Reply:
Let us keep this `val`, otherwise it will complain unused warning, maybe we will use this cfg for the SQL template in the future to avoid binary incompatibility.